lib/recommendation-server.js

```diff
@@ -46,7 +46,8 @@ RecommendationServer.prototype = {
     this.events.unsubscribe('printable-key', this.search);
   },
   _url: function(query) {
-    return `${this.server}/?q=${encodeURIComponent(query)}`;
+    const server = this.prefs.getCharPref('server');
```

Comment:
Switching to inline options, I'm just going to use a `<menulist>` to prevent this.